Tests/DatabaseTests/DatabaseTestCase.swift

```diff
@@ -1,41 +0,0 @@
-import Database
-import GitHub
-import GitHubTestSupport
-import Logging
-import ModelsTestSupport
-import Prelude
-import SnapshotTesting
-import XCTest
-
-class DatabaseTestCase: XCTestCase {
```

Comment:
I moved this back to `PointFreeTests`. We need to rely on `Current` for the env vars.

Reply:
why was this necessary? are env vars handled differently for GH actions?

Reply:
We weren't previously running CI under docker, so we were relying on a hard-coded localhost-based PostgreSQL server running.